pkg/reconciler/v1alpha1/revision/revision_test.go

```diff
@@ -563,11 +560,11 @@ func TestIstioOutboundIPRangesInjection(t *testing.T) {
 	want = "10.240.10.0/14"
 	annotations = getPodAnnotationsForConfig(t, "", want)
 	if got := annotations[resources.IstioOutboundIPRangeAnnotation]; got != want {
-		t.Fatalf("%v annotation = %v; want: %v", resources.IstioOutboundIPRangeAnnotation, got, want)
+		t.Fatalf("%v annotation is expected to have %v but got %v", resources.IstioOutboundIPRangeAnnotation, want, got)
 	}
 	annotations = getPodAnnotationsForConfig(t, "10.10.10.0/24", want)
 	if got := annotations[resources.IstioOutboundIPRangeAnnotation]; got != want {
-		t.Fatalf("%v annotation = %v; want %v", resources.IstioOutboundIPRangeAnnotation, got, want)
+		t.Fatalf("%v annotation is expected to have %v but got %v", resources.IstioOutboundIPRangeAnnotation, want, got)
```

Comment:
This undid a bunch of @vagababov changes

Reply:
I see, it undid the whole change because of the data racing.

Reply:
Those changes + my changes = data race:

https://gubernator.knative.dev/build/knative-prow/pr-logs/pull/knative_serving/3148/pull-knative-serving-unit-tests/1095417223662538752/

I was able to repro the flakiness with @vagababov 's PR but not with mine, so this feels like the lesser of two evils. I'm not sure how to go about untangling the data race with the k8s fakes.